src/__tests__/incidents/Incidents.test.tsx

```diff
@@ -31,28 +31,27 @@ describe('Incidents', () => {
       components: { sidebarCollapsed: false },
     } as any)
 
-    // eslint-disable-next-line react/prop-types
-    const Wrapper: React.FC = ({ children }) => (
+    return render(
       <Provider store={store}>
         <MemoryRouter initialEntries={[path]}>
-          <titleUtil.TitleProvider>{children}</titleUtil.TitleProvider>
+          <titleUtil.TitleProvider>
+            <Incidents />
+          </titleUtil.TitleProvider>
         </MemoryRouter>
-      </Provider>
+      </Provider>,
     )
-
-    return render(<Incidents />, { wrapper: Wrapper })
   }
 
   describe('routing', () => {
     describe('/incidents/new', () => {
       it('The new incident screen when /incidents/new is accessed', () => {
         setup([Permissions.ReportIncident], '/incidents/new')
-        expect(screen.getAllByText(/incidents.reports/i)[0]).toBeInTheDocument()
+        expect(screen.getByRole('form')).toBeInTheDocument()
```

Comment:
hm, not using the new `Report Incident form` label?

Reply:
🤦‍♂️ Totally forgot the aria-label value can be used via `{name: ...}`.
Will add that